.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could we add .idea to .gitignore, or is having this in the repo helpful?

Reply:
that's a good idea